vstudio/reinstall.ps1

```diff
@@ -31,8 +31,10 @@ if ($status -eq $false) {
 }
 
 & $devconBin remove "root\wnbd"
+wevtutil um wnbdevents.xml
```

Comment:
Let's add the "$scriptLocation\" prefix. This script may be called from somewhere else. We might define a path such as "$wnbdEvents" and add it to line 8.

Reply:
Whoops.